pkg/controller/discovery/web/web.go

```diff
@@ -343,7 +345,12 @@ func (h *BaseHandler) allow(sar auth.SelfSubjectAccessReview) int {
 		Log.Trace(err)
 		return http.StatusInternalServerError
 	}
-	restClient, err := apiutil.RESTClientForGVK(gvk, false, restCfg, codec)
+	c, err := rest.HTTPClientFor(restCfg)
```

Comment:
We have to supply an http client now, so this seems sane

Reply:
Yup, I came to the same conclusion